src/main/java/com/ericsson/ei/controller/FailedNotificationControllerImpl.java

```diff
@@ -35,16 +30,19 @@
 import com.ericsson.ei.utils.ResponseMessage;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
 /**
  * This class represents the REST GET mechanism to extract the aggregated data on the basis of the SubscriptionName from the Missed Notification
  * Object.
  */
 @Component
 @CrossOrigin
-@Api(value = "queryMissedNotification", tags = {"Missed notifications"})
-public class QueryMissedNotificationControllerImpl implements QueryMissedNotificationController {
+@Api(value = "failedNotification", tags = { "Failed notifications" })
```

Comment:
Same comment here, regarding spinal-case instead of camelCase

Reply:
done